.evergreen/config.yml

```diff
@@ -1882,6 +1882,16 @@ axes:
           GO_DIST: "/opt/golang/go1.15"
           PYTHON3_BINARY: python3
 
+  - id: ocsp-rhel-70
```

Comment:
Can you add a comment noting why this special variant is needed (due to SERVER-51364)?

Reply:
Yep, done.